activerecord/lib/active_record/migration.rb

```diff
@@ -791,6 +791,25 @@ def disable_ddl_transaction # :nodoc:
       self.class.disable_ddl_transaction
     end
 
+    ##
+    # :singleton-method: verbose
+    #
+    # By default, migrations will describe the actions they are taking, writing
+    # them to the console as they happen, along with benchmarks describing how
+    # long each step took.
+    #
+    # You read that setting through <tt>ActiveRecord::Migration.verbose</tt>.
```

Comment:
This has already been merged, but I had a pending review where I wanted to mention that we try to [avoid](https://edgeguides.rubyonrails.org/api_documentation_guidelines.html#pronouns) using "you" in the API docs.

I think we could rewrite it to describe what setting does (maybe somewhat similar to [verbose_query_logs](https://edgeapi.rubyonrails.org/classes/ActiveRecord.html#method-c-verbose_query_logs)):
```suggestion
    # Specifies if migrations will write the actions they are taking to the console as they
    # happen, along with benchmarks describing how long each step took. Defaults to
    # true.
```

I'm also not sure what we should do for class accessors. Describing both the reader and writer method or maybe have a special section for class accessors (this isn't supported by RDoc/SDoc yet).

Reply:
@p8 thank you for the feedback. I've opened https://github.com/rails/rails/pull/51296 to incorporate it.